docs/LTS.md

```diff
@@ -13,7 +13,15 @@ out in this document:
    [https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
 
 1. Major releases will receive security updates for an additional six months
-   from the release of the next major release.
+   from the release of the next major release. After this period is expired,
+   we will still review and release security fixes as long as they are
+   provided by the community and they do not violate other constrains,
+   e.g. minimum supported Node.js version.
+
+1. Major release will be tested and verified against all Node.js
+   releases that are supported by the
+   [Node.js LTS policy](https://github.com/nodejs/Release) within the
+   LTS period of that given Fastify release line.
```

Comment:
LTS period of `a` Fastify release line

(sounds simpler I think)

Reply:
The "given" clause is necessary to indicate it is specific to the LTS Fastify release.